lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -123,6 +126,8 @@ sub group_overview {
     $self->stash('only_tagged',     $only_tagged);
     $self->stash('comments',        \@comments);
     $self->stash('pinned_comments', \@pinned_comments);
+    $self->stash('latest_comment',  $latest_comment);
+    $self->stash('show_comments',   $show_comments);
```

Comment:
As this is the only use of these variables I suggest to directly query the params method here, i.e.

```perl
$self->stash('latest_comment', $self->param('latest_comment'));
…
```